doc/GettingStarted.md

```diff
@@ -9,6 +9,9 @@ This tutorial will walk through how to create an Uno application with the `dotne
 
 ### 1. Installing extension templates
 
+> [!NOTE]
+> Make sure to setup your environment first by [following our instructions](xref:Uno.GetStarted.vs2022).
+
 The `dotnet` templates included in the `Uno.Templates` package are used to easily create new projects that already reference the Uno.Extensions.
 
 * Open a command prompt and run the following:
```

Comment:
@jeromelaban I think this first section can be removed for the most part and point to the [dotnet new templates for Uno Platform](https://unoplatformdocstaging.z13.web.core.windows.net/articles/get-started-dotnet-new.html?tabs=windows) documentation right?
___
Also we were thinking with @dansiegel  and maybe it would make sense long term to have the template-related doc in the [uno.templates](https://github.com/unoplatform/uno.templates) repo